app/assets/stylesheets/mtl/extend/_avatars.scss

```diff
@@ -1,24 +1,26 @@
 // avatars
 .avatar,
 .mtl-layout-default .side-nav .avatar {
-  display: block;
-  width: 52px;
-  height: 52px;
-  line-height: 52px;
-  padding: 0;
-  border-radius: 26px;
+  display: inline-block;
+  border-radius: 50%;
+  height: 50px;
```

Comment:
@marcopluess what kind of avatar sizes do we actually need, also do we need additional classes like avatar-small or avatar-large?